starter-core/src/main/java/io/micronaut/starter/feature/other/OpenApiAdoc.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.other;
+
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.build.dependencies.MicronautDependencyUtils;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+import io.micronaut.starter.feature.FeatureContext;
+import io.micronaut.starter.feature.server.MicronautServerDependent;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class OpenApiAdoc implements Feature, MicronautServerDependent {
+
+    public static final String NAME = "openapi-adoc";
+
+    private static final Dependency DEPENDENCY_OPENAPI_ASCIIDOC = MicronautDependencyUtils
+            .openapi()
+            .artifactId("micronaut-openapi-adoc")
+            .annotationProcessor()
+            .build();
+
+    private final OpenApi openApiFeature;
+
+    public OpenApiAdoc(OpenApi openApiFeature) {
+        this.openApiFeature = openApiFeature;
+    }
+
+    @Override
+    public String getName() {
+        return NAME;
+    }
+
+    @Override
+    public String getTitle() {
+        return "OpenAPI Conversion to Asciidoc";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Adds and enables document conversion to asciidoc";
+    }
+
+    @Override
+    public void apply(GeneratorContext generatorContext) {
+        generatorContext.getConfiguration().put("micronaut.openapi.adoc.enabled", true);
```

Comment:
remove this property. The default value is `true`. Thus, we don't need to add it.